src/commcare_cloud/ansible/deploy_prometheus.yml

```diff
@@ -0,0 +1,41 @@
+- name: Encrypt Data Drive
+  hosts: control1
+  become: true
+  roles:
+    - {role: ecryptfs, tags: 'ecryptfs'}
+
+- name: Prometheus
+  hosts: prometheus
+  become: true
+  tasks:
+    - include_role:
+        name: cloudalchemy.prometheus
+    - import_role:
```

Comment:
why is one of these `include_role` and the other `import_role`? I think `import_role` is generally preferable except in specific scenarios where you're doing more dynamic things